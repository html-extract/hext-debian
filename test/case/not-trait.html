<html>
  <body>
    <div class="non-empty1">
      <div class="empty1"></div>
      <div class="non-empty2">
        <div class="empty2"></div>
        <div class="empty3"></div>
      </div>
    </div>
  </body>
</html>
