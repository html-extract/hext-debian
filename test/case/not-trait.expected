{"class":"non-empty1"}
{"class":"non-empty2"}
